omod/src/main/resources/config.xml

```diff
@@ -6,153 +6,255 @@
 	<id>${project.parent.artifactId}</id>
```

Comment:
It appears to me like you formatted this file, please remove any formatting otherwise I can't see what you added or removed?